README.md

```diff
@@ -11,29 +8,21 @@ this container and describe how to call BAMStats for the community.
 
 This tool has been validated as a CWL draft-3 and v1.0 CommandLineTool. 
 
-Versions that we tested with are the following 
-```
-avro (1.8.1)
-cwl-runner (1.0)
-cwl-upgrader (0.1.1)
-cwltool (1.0.20190915164430)
-schema-salad (4.5.20190906201758)
-setuptools (25.1.6)
-```
+Versions that we tested with are documented iin https://raw.githubusercontent.com/dockstore/dockstore/1.13.0-beta.3/dockstore-webservice/src/main/resources/requirements/1.13.0/requirements3.txt
 
 
 ## Building Manually
 
-Normally you would let [Quay.io](http://quay.io) build this.  But, if you need to build
+Normally you would let GitHub actions build this.  But, if you need to build
 manually you would execute:
 
-    docker build -t collaboratory/dockstore-tool-bamstats:1.25-7 .
+    docker build -t ghcr.io/dockstore/dockstore-tool-bamstats:1.25-8 .
```

Comment:
I recently discovered that if you host the image on ghcr.io, this will trigger an error in Cromwell (and by extension the Dockstore CLI -- can confirm I get the error with this URL in the latest stable version of the Dockstore CLI). 

https://github.com/broadinstitute/cromwell/issues/6827

Reply:
See github action, error seems like more of a `warn`/warning that we can ignore 

```
[2022-08-11 17:24:51,73] [warn] BackendPreparationActor_for_e2a7a05d:bamstatsWorkflow.bamstats:-1:1 [e2a7a05d]: Docker lookup failed
java.lang.Exception: Registry ghcr.io is not supported
```

Reply:
I believe it's not acceptable to present to the user with an unclear warning on a tutorial repository. We should really consider moving this to Docker Hub.